src/extension/tab/hook.ts

```diff
@@ -223,28 +214,6 @@ tab.on("explorerRequest", (message) => {
   }
 });
 
-/**
- * Attempt to find the client on a 1-second interval for 10 seconds max
- */
-let interval: NodeJS.Timeout;
-function findClient() {
```

Comment:
We no longer need this at all since we send a `registerClient` message any time the client is registered which will connect it for us.